pkgs/by-name/pa/paper-plane/tdlib.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchFromGitHub, gperf, openssl, readline, zlib, cmake, lib, stdenv }:
```

Comment:
this looks like a copy and paste of ./pkgs/development/libraries/tdlib/default.nix

why not just override the version ?

Reply:
This is probably not a good idea. For example, workarounds may be added after tdlib is updated, but these operations are not applicable to older versions, causing an increase in maintenance costs.

Reply:
one could argue same for duplication, there can be a change in dependency eg `gperf openssl readline zlib` or `gcc` that will require fixes to both copies of tdlib

maybe others can weigh in ? cc @SuperSandro2000 @mweinelt

Reply:
The dependencies of the new version are not necessarily related to the dependencies of the old version. If we need to reuse these dependencies, more problems may be introduced. Likewise, meta.licenses are not necessarily consistent.

Reply:
paper-tdlib and nixpkgs tdlib both use same versions of `gperf openssl readline zlib`
if say, we new openssl is incompatible, then both tdlib will need to be patched. Thats `an increase in maintenance costs.` 

as for license, if it changed, then in override license should also be overridden

I think only when there is sufficient differences between versions copy may be warranted. Currently there are no differences apart from src

Reply:
I have changed to overrideAttrs, but if the build fails because the original package modifies the build script someday, I will not try to maintain this solution.